app/controllers/api/v01/bibs_controller.rb

```diff
@@ -137,7 +137,7 @@ def delete_teacher_sets
 
     saved_teacher_sets = []
     @request_body.each do |teacher_set_record|
-      # Delete teacher-set record by bib_id
+      # Get teacher-set record by bib_id
```

Comment:
how come this change?  it looks like the teacher set is still being destroyed, is there other stuff that happens?

Reply:
@daryanypl I updated little more document